push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/PushyApnsSender.java

```diff
@@ -214,9 +214,17 @@ private ApnsClient buildApnsClient(final iOSVariant iOSVariant) {
                 final ApnsClientBuilder builder = new ApnsClientBuilder();
                 builder.setClientCredentials(stream, iOSVariant.getPassphrase());
 
-                if (ProxyConfiguration.hasSocks()) {
-                    final Proxy proxy = ProxyConfiguration.socks();
-                    builder.setProxyHandlerFactory(new Socks5ProxyHandlerFactory(proxy.address()));
+                if (ProxyConfiguration.hasProxyConfig()) {
```

Comment:
@PhilipGough with `hasProxyConfig` it sounds like a more generic question about proxy, but looking at the block of code below, I think it is really about `hasHttpProxyConfig`, right ?